### PR TITLE
Adding TARS related resources

### PR DESCRIPTION
**Pull request by kishaloyhalder**

### CODE REVIEW

**Review by alanakbik — COMMENTED** [review]
Looks good, just a few minor comments!

File: README.md [hunk 1]
```diff
@@ -123,6 +123,7 @@ We provide a set of quick tutorials to get you started with the library:
 * [Tutorial 6: Loading a Dataset](/resources/docs/TUTORIAL_6_CORPUS.md)
 * [Tutorial 7: Training a Model](/resources/docs/TUTORIAL_7_TRAINING_A_MODEL.md)
 * [Tutorial 8: Training your own Flair Embeddings](/resources/docs/TUTORIAL_9_TRAINING_LM_EMBEDDINGS.md)
+* [Tutorial 9: Training a Zero Shot Text Classifier](/resources/docs/TUTORIAL_10_TRAINING_ZERO_SHOT_MODEL.md)
```
[the remark this answers is not in the document]
Comment (kishaloyhalder) [reply README.md:126]: Added a note in the landing page in the latest commit.

File: flair/models/text_classification_model.py [hunk 2]
```diff
@@ -492,3 +504,402 @@ def __str__(self):
...
+    static_label_type = "tars_label"
+    static_adhoc_task_identifier = "adhoc_dummy"
+
+    def __init__(
+            self,
+            task_name: str,
+            label_dictionary: Dictionary,
+            batch_size: int = 16,
+            document_embeddings: str = 'bert-base-uncased',
+            num_negative_labels_to_sample: int = 2,
```
[the remark this answers is not in the document]
Comment (kishaloyhalder) [reply flair/models/text_classification_model.py:531]: Thanks for pointing it out. Have added some explanation about this parameter.

File: flair/models/text_classification_model.py [hunk 3]
```diff
@@ -492,3 +504,402 @@ def __str__(self):
...
+    static_label_no = "NO"
+    static_label_type = "tars_label"
+    static_adhoc_task_identifier = "adhoc_dummy"
+
+    def __init__(
+            self,
+            task_name: str,
+            label_dictionary: Dictionary,
+            batch_size: int = 16,
+            document_embeddings: str = 'bert-base-uncased',
```
[the remark this answers is not in the document]
Comment (kishaloyhalder) [reply flair/models/text_classification_model.py:530]: Most of the users are probably going to use the TextClassifier class as is.

**Comment by alanakbik** [comment]
@kishaloyhalder @yosipk thanks this looks great - I can't wait to see what the community does with this new feature!

**Pull request merged**